.scrutinizer.yml

```diff
@@ -135,6 +135,8 @@ build:
     dependencies:
         before:
             - composer global require "fxp/composer-asset-plugin:^1.3.1"
+            - composer require "codemix/yii2-streamlog:^1.3"
+            - composer require "silinternational/email-service-php-client:^2.0.0"
             - composer install
```

Comment:
This `composer install` command [will also install the dev dependencies](https://getcomposer.org/doc/03-cli.md#install-i), so let's put these optional dependencies in the composer.json's "require-dev" section rather than defining them here in `.scrutinizer.yml`.

I think it will be a lot more maintainable that way, since it's not a common practice for us to know that any time we change composer dependency information we should also look in the .scrutinizer.yml file.